src/components/Card/CartCard.js

```diff
@@ -0,0 +1,49 @@
+import "./ProductCard.css";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
```

Comment:
You can create an index.js file inside context folder and export all the contexts from it. This way you will be able to import all contexts from a single file.